http/src/main/scala/org/apache/pekko/http/javadsl/unmarshalling/Unmarshaller.scala

```diff
@@ -14,21 +14,20 @@
 package org.apache.pekko.http.javadsl.unmarshalling
 
 import java.util.concurrent.CompletionStage
+import java.util.Optional
 
 import org.apache.pekko
 import pekko.actor.ClassicActorSystemProvider
 import pekko.annotation.InternalApi
 import pekko.http.impl.model.JavaQuery
 import pekko.http.impl.util.JavaMapping
 import pekko.http.impl.util.JavaMapping.Implicits._
-import pekko.http.javadsl.model._
+import pekko.http.{ javadsl => jm }
+import jm.model._
 import pekko.http.scaladsl.model.{ ContentTypeRange, ContentTypes }
 import pekko.http.scaladsl.unmarshalling
 import pekko.http.scaladsl.unmarshalling.FromEntityUnmarshaller
-import pekko.http.scaladsl.unmarshalling.Unmarshaller.{
-  EnhancedFromEntityUnmarshaller,
-  UnsupportedContentTypeException
-}
```

Comment:
This will break user's code

Reply:
How, it passes MiMa? The exception being thrown here is the same.